Seminars/Sem.01/Pract.01/Task04.cpp

```diff
@@ -1,30 +1,53 @@
-#include <iostream>
-
-using std::cin, std::cout, std::endl;
-
-const int nameSize = 30;
-
-struct Student {
-    char name[nameSize + 1];
-    unsigned int facultyNumber;
-};
-
-void initStudent(Student &student) {
-    cout << "Enter name: ";
-    cin.getline(student.name, 31);
-    cout << "Enter faculty number: ";
-    cin >> student.facultyNumber;
-    cin.ignore();
-}
-
-void printStudent(const Student &student) {
-    cout << "Name: " << student.name << endl;
-    cout << "Faculty number: " << student.facultyNumber << endl;
-}
-
-int main() {
-    Student student{};
-    initStudent(student);
-    printStudent(student);
-    return 0;
-}
+#include <iostream>
+
+const int NAME_SIZE = 30;
+
+struct Student
+{
+	char name[NAME_SIZE + 1];
+	unsigned int fn;
+};
+
+void initSTudent(Student& student)
+{
+	std::cout << "Enter name: ";
+	std::cin.getline(student.name, NAME_SIZE + 1);

```

Comment:
Read documentation about getline
(The ending null character that signals the end of a c-string is automatically appended to s after the data extracted.)